src/utils/route/routeParser.ts

```diff
@@ -54,9 +54,31 @@ export class RouteParser {
 		route.name = routeName.text;
 		route.pos = this.tsParser.sourceFile.getLineAndCharacterOfPosition(routeName.pos);
 		route.path = this.tsParser.path;
+		route.views = [];
 
 		route.template = this.templateParser.createTemplate(config, this.tsParser);
 
+		const viewsNode = config.get('views') as ts.ObjectLiteralExpression;
+
+		if (viewsNode !== undefined) {
+			const viewConfigNodes = new ConfigParser(viewsNode).entries();
+
+			for (const viewNode of viewConfigNodes) {
+				const viewRoute = new Route();
+				viewRoute.name = `${route.name}[${viewNode[0]}]`;
+				viewRoute.pos = this.tsParser.sourceFile.getLineAndCharacterOfPosition(viewNode[1].pos);
+				viewRoute.path = this.tsParser.path;
+
+				if (ts.isStringLiteral(viewNode[1])) {

```

Comment:
This is nice. I didn't know such type guards exist in TS library. This seems better than my implementation of `isTsKind`. Keep using it. You can start replacing `isTsKind` with this wherever you're changing anything. I like it very much, it's more concise and self documenting at once.